src/routes/Routes.tsx

```diff
@@ -145,6 +154,28 @@ const Routes = () => {
             hasBackLink: false,
           }}
         />
+        <WalletRequiredRoute
+          isConnected={isConnected}
+          didClaimProfile={didClaimProfile}
+          path={UNWRAP_TOKEN_ROUTE}
+          component={UnwrapTokensPage}
+          layout={NavBar}
+          routeProps={{
+            backText: MSG.backToMetacolony,
+            backRoute: `/colony/${METACOLONY_ENS}`,
```

Comment:
These should still be colony generic, and not specific to the metacolony, as we might want to open up this to all colonies at some point.

Plus, you already test this inside the route and redirect if it's a different colony, so the "safety" catch is there